app/models/ngo.rb

```diff
@@ -2,4 +2,12 @@ class Ngo < ApplicationRecord
     # Associations 
     belongs_to :user 
     has_many :donation_requests
+
+    # NGO registration validations
+    validates :organization_name, presence: true, uniqueness: true
+    validates :address, presence: true
+    validates :organization_phone_number, presence: true, uniqueness: true, length: {minimum: 10, maximum: 13}
```

Comment:
reminder to check on this code for better validation of organization_phone_number
and make sure the error displays well

Reply:
Okay, remind me when I miss it tomorrow